stream/src/main/scala/org/apache/pekko/stream/impl/JsonObjectParser.scala

```diff
@@ -115,7 +115,9 @@ import pekko.util.ByteString
   private def seekObject(): Boolean = {
     completedObject = false
     val bufSize = buffer.length
-
+    if (bufSize == 0) {
```

Comment:
I don't like 'return' statements. Can't you just do this ?

```
if (bufSize > 0) {
  // add everything from the rest if the function here
}
```

Reply:
Agreed, maybe not super important in that case, but let's avoid a potential exception being thrown.